.changelog/1732034799.md

```diff
@@ -0,0 +1,28 @@
+---
+applies_to: ["client"]
```

Comment:
nit: given code changes are mostly under `aws`, the release note should probably show up in a `aws-sdk-rust` release. This should be either `["client", "aws-sdk-rust"]` or just `["aws-sdk-rust"]`